spec.html

```diff
@@ -25907,30 +26113,44 @@ <h1>Static Semantics: ModuleRequests ( ): a List of Strings</h1>
         </emu-alg>
         <emu-grammar>ModuleItemList : ModuleItemList ModuleItem</emu-grammar>
         <emu-alg>
-          1. Let _moduleNames_ be ModuleRequests of |ModuleItemList|.
-          1. Let _additionalNames_ be ModuleRequests of |ModuleItem|.
-          1. For each String _name_ of _additionalNames_, do
-            1. If _moduleNames_ does not contain _name_, then
-              1. Append _name_ to _moduleNames_.
-          1. Return _moduleNames_.
+          1. Let _requests_ be ModuleRequests of |ModuleItemList|.
+          1. Let _additionalRequests_ be ModuleRequests of |ModuleItem|.
+          1. For each ModuleRequest Record _mr_ of _additionalRequests_, do
+            1. If _requests_ does not contain a ModuleRequest Record _mr2_ such that ModuleRequestsEqual(_mr_, _mr2_) is *true*, then
+              1. Append _mr_ to _requests_.
+          1. Return _requests_.
         </emu-alg>
         <emu-grammar>ModuleItem : StatementListItem</emu-grammar>
         <emu-alg>
           1. Return a new empty List.
         </emu-alg>
         <emu-grammar>ImportDeclaration : `import` ImportClause FromClause `;`</emu-grammar>
         <emu-alg>
-          1. Return ModuleRequests of |FromClause|.
+          1. Let _specifier_ be SV of |FromClause|.
```

Comment:
```suggestion
          1. Let _specifier_ be the SV of |FromClause|.
```

We don't actually want to use this convention in the future, but we're consistent about using it currently, and you should at least be consistent about it within this PR.